app/views/govuk_publishing_components/components/_related_navigation.html.erb

```diff
@@ -2,7 +2,7 @@
 <% related_content = related_nav_helper.related_navigation %>
 <% if related_content.map(&:values).flatten.any? || related_nav_helper.other.flatten.any? %>
 <% random = SecureRandom.hex(4) %>
-  <aside class="gem-c-related-navigation" role="complementary">
+  <div class="gem-c-related-navigation">
```

Comment:
If there's anything interesting could we put it in the accessibility acceptance criteria so that someone trying to be useful (me) later does not add aside back?

Reply:
Good shout Nick, I've added a note about it in the documentation.

Reply:
Thanks Emma 👍